provisioning_templates/snippet/redhat_register.erb

```diff
@@ -178,7 +179,7 @@ snippet: true
   <% if host_param('subscription_manager_repos') %>
     # workaround for RHEL 6.4 bug https://bugzilla.redhat.com/show_bug.cgi?id=1008016
     subscription-manager repos --list > /dev/null
-    <%= "subscription-manager repos --enable #{host_param('subscription_manager_repos').gsub(',', ' --enable')}" %>
+    <%= "subscription-manager repos --enable #{host_param('subscription_manager_repos').gsub(',', ' --enable ')}" %>
```

Comment:
I don't know if this works, but maybe the replacement should consume all whitespace as well? Just a visual thing though.
```suggestion
    <%= "subscription-manager repos --enable #{host_param('subscription_manager_repos').gsub(/,\s*/, ' --enable ')}" %>
```

Reply:
Yep, makes it slightly better to read in case you need to debug the rendered template.
I tested this in my instances (1.21.4 + 1.23.0) and it works.

I will update my PR to include this change.